### ISSUE TITLE
Objects support

### ISSUE THREAD

**Issue by ReshiramZekrom1** [boorzz/samp-opws-fix#1]
This is dope! Will be any objects support in the future?

**Comment by boorzz** [comment]
> This is dope!
Thank you! <3

> Will be any objects support in the future?
What do you mean by "object support"?

**Comment by ReshiramZekrom1** [comment]
> > This is dope!
> 
> Thank you! <3
> 
> > Will be any objects support in the future?
> 
> What do you mean by "object support"?

The function is working flawlessly when I shoot players or vehicles but it's simply not called when I shoot server sided Objects. Didn't try the ones created directly with CreateObject but instead the CreateDynamicObject ones which rely on Player Objects, tho

**Comment by boorzz** [comment]
That's weird. I tried to shoot Streamer (player objects) and it does trigger the callback. I've recently made an update to the include, can you confirm you're using the latest version?

![Screenshot 2020-12-24 141720](https://user-images.githubusercontent.com/74167411/103101121-d6063580-45f4-11eb-90b3-791dc661b08d.png)

**Comment by ReshiramZekrom1** [comment]
This is kinda awkward, but I've simply put the "OnPlayerShootDynamicObject" callback in my script and now everything is working as intended 😅 Just for testing purposes, I removed that callback and OnPlayerWeaponShot started again to don't be even called when I shoot Dynamic objects. Tested what happens when I shoot object created with CreateObject and OPWS works absolutely fine, even if I don't put "OnPlayerShootDynamicObject" in the script. End of the story: everything is now working perfectly, really, just wanted to report this bug which is totally harmless. Thank you again for this work, I always wanted to use the OPWS features in a lagshot server!
